frontend/src/pages/EndRoundPage/index.tsx

```diff
@@ -28,11 +28,11 @@ const EndRoundPage = ({ roundLimit }: { roundLimit: number }) => {
       <div className={styles.endOfRound}>
         <div style={{ margin: `24px 0` }}>
           <h5>The Setup:</h5>
-          <h2>{setup.setup}</h2>
+          <h2>{setup?.setup}</h2>
```

Comment:
What happens if this is actually null? I notice you handle these cases where you make nullability changes in other parts of the code but not here

Reply:
So with the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) it returns `undefined` which is just rendered as blank by React - realistically we should probably look into some more advanced handling of these edge-cases down the line but for now I'm happy that this is good enough

Reply:
Alright - I guess that's relatively graceful. Down the line better checking would be good but not a must